app/controllers/account/billing-info/invoices.js

```diff
@@ -0,0 +1,8 @@
+import Controller from '@ember/controller';
+import { filterBy } from '@ember/object/computed';
+
+export default Controller.extend({
```

Comment:
Lets start moving to ES6 style classes for all new resources. 
https://github.com/fossasia/open-event-frontend/pull/3169/files